CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 
     When the new `active_support_custom_events_names` configuration parameter is set equal to an array of custom event names to subscribe to, the agent will now subscribe to each of the names specified and report instrumentation for the events when they take place. [Creating custom events](https://guides.rubyonrails.org/active_support_instrumentation.html#creating-custom-events) is simple and now reporting instrumentation for them to New Relic is simple as well. [PR#1659](https://github.com/newrelic/newrelic-ruby-agent/pull/1659)
 
+  * **Bugfix: Support older versions of the redis-client gem, handle unknown Redis database index
```

Comment:
I think we generally capitalize the gem name, but I'd be open to changing this. In 8.13.1 `redis-client` was Redis Client. 

```suggestion
  * **Bugfix: Support older versions of the Redis Client gem, handle unknown Redis database index
```

Reply:
In this context we have fully 5 names to keep track of:

- "Redis", the name of the datastore software itself
- "redis-rb", the project name for the "redis" gem
- "redis", a long running Ruby gem for Redis
- "RedisClient", the project name for the "redis-client" gem
- "redis-client", a relatively new Ruby gem for Redis

My intention was to disambiguate the gem names from the "Redis" datastore software name.

Frustratingly, we seem to have never referred the "redis-rb" project correctly as either "redis-rb" or "redis", opting instead for the confusing "Redis". As for "RedisClient", we've tossed a space into that one and gone with "Redis Client" for v8.13.1.

I'll go ahead and use "redis-rb" and "RedisClient" for v8.13.1 and v8.14.0 and leave the older changelog entries alone. I'm okay with using project names instead of gem names, but not capitalizing them unless the project namer has done so.

Reply:
Updated with [772de0a](https://github.com/newrelic/newrelic-ruby-agent/pull/1673/commits/772de0ad48a8e280953684ff81dba2ff1c344ea0)